pkg/server/router/webhook.go

```diff
@@ -0,0 +1,174 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	"github.com/tbd54566975/ssi-service/pkg/server/framework"
+	svcframework "github.com/tbd54566975/ssi-service/pkg/service/framework"
+	"github.com/tbd54566975/ssi-service/pkg/service/webhook"
+)
+
+type WebhookRouter struct {
+	service *webhook.Service
+}
+
+func NewWebhookRouter(s svcframework.Service) (*WebhookRouter, error) {
+	if s == nil {
+		return nil, errors.New("service cannot be nil")
+	}
+	webhookService, ok := s.(*webhook.Service)
+	if !ok {
+		return nil, fmt.Errorf("could not create webhook router with service type: %s", s.Type())
+	}
+	return &WebhookRouter{service: webhookService}, nil
+}
+
+type CreateWebhookRequest struct {
+	WebhookNoun   webhook.Noun   `json:"webhookNoun" validate:"required"`
```

Comment:
Here and elsewhere. 

Can you add documentation so the public facing docs have a description for each field (hopefully with and example :) ?

See credentialAPI for an example of how the documentation is propagated.

Reply:
added comments to the request and other areas